packages/dev/core/src/Animations/animatable.ts

```diff
@@ -26,6 +27,12 @@ export class Animatable {
     private _frameToSyncFromJump: Nullable<number> = null;
     private _goToFrame: Nullable<number> = null;
 
+    /** Order of play so that this animatable is played before all other animatables */
+    public static readonly PLAYORDER_FIRST = Constants.ANIMATABLE_PLAYORDER_FIRST;

```

Comment:
Should "play order" replaced by "playback order" everywhere?

Reply:
cc @bghgary as he is the only one that would know :-)

Reply:
Play order is okay with me.